packages/core/src/index.ts

```diff
@@ -13,7 +13,7 @@ export {
   setUser,
   withScope,
 } from '@sentry/minimal';
-export { addGlobalEventProcessor, getCurrentHub, getHubFromCarrier, Hub, makeMain, Scope } from '@sentry/hub';
+export { addGlobalEventProcessor, getCurrentHub, getHubFromCarrier, Hub, makeMain, Scope, Session } from '@sentry/hub';
```

Comment:
Do we really need to export `Session` from `@sentry/core`? The core is growing out of control 😄 

---

I see we use this to make a minimal change in imports in `@sentry/browser`. I wonder if we could just import from `@sentry/hub` in `@sentry/browser`? I'd love to see a picture of the current dependency graph if we have one!